LookingGlass.php

```diff
@@ -428,7 +428,11 @@ public static function getLatency(): float
      */
     private static function getLatencyFromSs(string $ip): array
     {
-        $lines = shell_exec('/usr/sbin/ss -Hti state established');
+        if (file_exists('/usr/sbin/ss')) {
+                $lines = shell_exec('/usr/sbin/ss -Hti state established'); // RHEL Directory for ss
+            } elseif (file_exists('/usr/bin/ss')) {
+                $lines = shell_exec('/usr/bin/ss -Hti state established'); // Debian Directory for ss
+            }
```

Comment:
Can you please fix the indentation?

```suggestion
            $lines = shell_exec('/usr/sbin/ss -Hti state established'); // RHEL Directory for ss
        } elseif (file_exists('/usr/bin/ss')) {
            $lines = shell_exec('/usr/bin/ss -Hti state established'); // Debian Directory for ss
        }
```